### DISCUSSION TITLE
Unable to import file more than 26 MB

### DISCUSSION THREAD

**Discussion by dheeban-k** [outline/outline#4185] [Self Hosting] [answered]
Hi @tommoor,

Hope you're doing well.
Bi-weekly i used to check import and export of my contents in outline,this time the export size was 32MB.
On trying to import, I face errors.

Need to know how to avoid this error of "File is too large to import" . Please help me with this, since we're rapidly growing our users files from notion and even files in outline cannot be imported in this way.
Attaching the error image and browser logs for reference.
Let me know if i need to attach any additional information. Thanks

<img width="1462" alt="image" src="https://user-images.githubusercontent.com/101546615/192745204-80636ecd-e12e-4f7c-93c2-4ce5d69f965a.png">

[scroll.zingworks.com-1664357430705.log](https://github.com/outline/outline/files/9663524/scroll.zingworks.com-1664357430705.log)

**Comment by ChetanGoti** [comment, marked as answer]
What is your max S3 upload size configuration value?

Default in [.env.sample](https://github.com/outline/outline/blob/main/.env.sample#L54) has value 26214400 bytes which is exactly 26.21MB

e.g:
```bash
AWS_S3_UPLOAD_MAX_SIZE=26214400
```

May be try making it larger as I guess import would also using this value.

**Reply by dheeban-k** [reply]
I just found it now. I made the changes, thanks for your quick response @ChetanGoti 
@tommoor awesome product mate :)
Cheers !

**Reply by dheeban-k** [reply]
> I just found it now. I made the changes, thanks for your quick response @ChetanGoti @tommoor awesome product mate :) Cheers !

The environment variable AWS_S3_UPLOAD_MAX_SIZE was earlier set to 26214400. Now increased the byte count, error fixed.
Thanks @ChetanGoti

**Comment by brightprogrammer** [comment]
Even though this as already been marked as answered. I'll upload my solution for anyone having a slightly different problem.

I selfhost my outline instance and use a reverse proxy to enable https and manage many other services at once. The upload will also fail if your reverse proxy denies upload of larger file sizes. I use nginx so I had to add 

```nginx
server {
    ...
    client_max_body_size 100M;  # Adjust the size as needed
    ...
}
```

To the server block handling the proxy for outline. This solved the issue.
